src/operator/skip.ts

```diff
@@ -8,8 +8,8 @@ import { TeardownLogic } from '../Subscription';
  *
  * <img src="./img/skip.png" width="100%">
  *
- * @param {Number} the `n` of times, items emitted by source Observable should be skipped.
- * @return {Observable} an Observable that skips values emitted by the source Observable.
+ * @param {Number} n - The number of times, items emitted by source Observable should be skipped.
```

Comment:
The `@param` needs to match the argument name in the function declaration so this should be:

@param {Number} total